### PR TITLE
Guideline for test and PR

### PR DESCRIPTION
**Pull request by esitsolf**
I created 2 guidelines.

1. Pr-guidelines: 

    - walkthrough the way to create a PR step by step
    - Give all the hints that could be needed to create a "good" PR
    - Show how to manage the changes requested

2. Test-procedure-guideline:

    - An unexhaustive list of the steps to be done to test appropriatly the app when doing the Review of a PR or preparing for a PR

***These are a living documents. They can be change as much as we want!***

### CODE REVIEW

**Comment by Vanessi-K** [comment]
From my side everything looks good.

File: pr-guideline.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  - Be open for feedback => no one wants to bother you
+  - Keep a critical eye => You know want you did, sometimes the feedback/opinion of the reviewer won't work on your specific code
+- Do the changes if needed
+- Commit and push again
+- Check that dev is still merge on your branch (no new accepted PR in between)
+- Ask for a new review (up right on the PR screen)
+
+## When the PR is accepted
+
+Go to the discord Channel and write a message @everyone and give a list of files that you changed.
```
Comment (Annacc211018) [inline pr-guideline.md:55]: the Pull Request Channel

File: test-procedure-guideline.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+9. Download your comic and verify that it looks as on the editor
+10. Save your comic as a draft
+11. Navigate back to the index
+12. Open the draft
+13. Modify the draft (redo points 2 to 6)
+14. Save the draft
+15. Navigate back to the index
+16. Delete the draft
+17. Create a new comic with more than 1 panel
+18. Redo points 2 to 16 (the assets should be on different panels)
```
Comment (Annacc211018) [inline test-procedure-guideline.md:35]: add as a last point to report issues on the PR - I mean it's kinda self explanatory but just for completion sake :)

**Review by Annacc211018 — COMMENTED** [review]
Covers everything important and is very understandable :) Thanks!

**Pull request merged**